docs/UserGuide.md

```diff
@@ -31,7 +32,7 @@ Argument List:
 - since : Optional. The start date of analysis. Format: `DD/MM/YYYY`
 - until : Optional. The end date of analysis. Format: `DD/MM/YYYY`
 - formats : Optional. The file formats to analyse. Formats: `alphanumerical file formats`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
-
+- start-server : Optional. The path to the report directory. The server will be started to display the dashboard.
```

Comment:
`-view`
This isn't `The path to the report directory`, but rather if we specify this flag **with** a path to the report directory, then a server will be started to display the dashboard.
Also, do mention that this will ignore all the other arguments given.